app/src/main/java/io/github/mattpvaughn/chronicle/features/player/PlayerExt.kt

```diff
@@ -24,3 +30,38 @@ fun Player.seekRelative(trackListStateManager: TrackListStateManager, durationMi
         seekTo(trackListStateManager.currentTrackIndex, trackListStateManager.currentTrackProgress)
     }
 }
+
+/** Skip to next chapter */
+fun Player.skipToNext(trackListStateManager: TrackListStateManager, currentlyPlaying: CurrentlyPlaying, progressUpdater: ProgressUpdater) {
+    Timber.i("Player.skipToNext called")
+    val currentChapterIndex = currentlyPlaying.book.value.chapters.indexOf(currentlyPlaying.chapter.value)
+    val nextChapterIndex = currentChapterIndex + 1
+    if(nextChapterIndex < currentlyPlaying.book.value.chapters.size) {
+        val nextChapter = currentlyPlaying.book.value.chapters[nextChapterIndex]
+        Timber.d("NEXT CHAPTER: index=$nextChapterIndex id=${nextChapter.id} trackId=${nextChapter.trackId}  offset=${nextChapter.startTimeOffset} title=${nextChapter.title}")
+        seekTo(trackListStateManager.currentTrackIndex, nextChapter.startTimeOffset)
```

Comment:
in a book where each chapter is its own track, this would seek to the start of the current track (since the next chapter will have an offset of 0 since it starts at 0 of it's own track). You can get the index of the chapter's track with something like this:

```
    val containingTrack = trackListStateManager.trackList
        .firstOrNull {
            it.id.toLong() == previousChapter.trackId
        }
    val containingTrackIndex = trackListStateManager.trackList.indexOf(containingTrack)

```